wiki/Welcome/fr.md

```diff
@@ -0,0 +1,47 @@
+# Bienvenue
+
+Bienvenue dans Osu!, un jeu de rythme gratuit développé par peppy avec quatre modes de jeu: osu! Standard, un simulateur de clic sur un cercle; osu! taiko, un émulateur de batterie; osu! catch, un récupérateur de salade de fruits; et osu! mania, un synthétiseur puissant.
```

Comment:
Keep names in their original format.

`Osu!` -> `osu!`
`osu! Standard` -> `osu!standard`
`osu! taiko` -> `osu!taiko`
`osu! catch` -> `osu!catch`
`osu! mania` -> `osu!mania`